gslib/tests/test_acl.py

```diff
@@ -850,11 +870,11 @@ def test_shim_translates_predefined_acl_set_object(self):
           'CLOUDSDK_ROOT_DIR': 'fake_dir',
       }):
         mock_log_handler = self.RunCommand(
-            'acl', ['set', 'private', 'gs://bucket/object'],
+            'acl', ['set', 'publicRead', 'gs://bucket/object'],
```

Comment:
Any reason for changing this to publicRead? Does not matter much but I think private should have been fine as well?

Reply:
`private` is the same string for XML and JSON, whereas hyphenated strings (e.g. `public-read`) show translation to JSON for gcloud storage.